src/include/platform/internal/GenericPlatformManagerImpl.cpp

```diff
@@ -134,8 +134,8 @@ CHIP_ERROR GenericPlatformManagerImpl<ImplClass>::_Shutdown()
     err = InetLayer().Shutdown();
 
 #if CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE
-    ChipLogError(DeviceLayer, "BLE layer shutdown");
-    err = BLEMgr().GetBleLayer()->Shutdown();
+    ChipLogError(DeviceLayer, "BLE shutdown");
+    err = BLEMgr().Shutdown();
```

Comment:
I have no idea why err value is lost here and above :( probably we should log errors at least, however I did not try to fix this in this PR.